hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestCOWDataSourceExtend.scala

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
we add a new UT file， since TestCowDataSource.scala is too large, add this ut to it will lead checkstyle failed!